x-pack/plugins/maps/public/classes/layers/blended_vector_layer/blended_vector_layer.ts

```diff
@@ -328,4 +329,11 @@ export class BlendedVectorLayer extends VectorLayer implements IVectorLayer {
 
     super._syncData(syncContext, activeSource, activeStyle);
   }
+
+  async getLicensedFeatures(): Promise<LICENSED_FEATURES[]> {
```

Comment:
Should this method remove any duplicates if _clusterSource and _documentSource returned the same features?

Reply:
In practice that will not happen, but looks better for completeness. Can address when used of licensed features becomes more complicated